vulnerabilities/importers/github.py

```diff
@@ -0,0 +1,342 @@
+# Copyright (c) 2017 nexB Inc. and others. All rights reserved.
```

Comment:
Nit: best to remove the date IMHO here or add 2020

Reply:
Done